portfolio/src/main/webapp/script.js

```diff
@@ -28,7 +28,23 @@
 //   greetingContainer.innerText = greeting;
 // }
 async function getDataUsingAsyncAwait() {
-  const response = await fetch('/data');
-  const data = await response.text();
-  document.getElementById('data-container').innerText = data;
+   //const response = await fetch('/data');
+   fetch('/data').then(response => response.json()).then((favorite) => {
+  //const data = await response.json(favorite);
+  const getParagraph = document.getElementById('data-container');
+  getParagraph.innerHTML=favorite;
```

Comment:
At this point favorite is an array of strings. Something like the following code can iterate over it:

```
favorite.forEach(fav => {
  getParagraph.appendChild(createParagraph(fav));
});
```

Once the functionality works, could you review the code to see if variable names are descriptive?